src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,578 @@
+tileShorthand:
+  title: 'Using Terragrunt configuratiions'
+  description: 'Learn how to use Terragrunt to manage configurations in multple environments'
+resources:
+  - title: 'Terraform New Relic Provider'
+    url: 'https://github.com/newrelic/terraform-provider-newrelic'
+  - title: 'New Relic documentation'
+    url: https://docs.newrelic.com
+  - title: 'Terraform documentation'
+    url: https://terraform.io/docs/index.html
+  - title: 'Terragrunt documentation'
+    url: https://terragrunt.gruntwork.io/docs/getting-started/quick-start/
+tags:
+  - notification channel
+  - alerts
+  - terraform
+---
+
+<Intro>
+
+[Terraform](https://www.terraform.io/) is a popular infrastructure-as-code software tool built by HashiCorp. You use it to provision all kinds of infrastructure and services, including New Relic dashboards and alerts.
+
+[Terragrunt](https://terragrunt.gruntwork.io/#:~:text=Terragrunt%20is%20a%20thin%20wrapper,modules%2C%20and%20managing%20remote%20state.) is a thin wrapper that provides extra tools for keeping your configurations DRY, working with multiple Terraform modules, and managing remote state.
+
+In this guide, you will use Terragrunt to generate your Terraform configurations, create files with Terragrunt, and reuse the Terragrunt configuration to manage multiple environments.
+
+<Video id="ivbp4e8t1f" type="wistia" />
+
+</Intro>
+
+## Before you begin
+
+To use this guide, you should have some basic knowledge of both New Relic and Terraform. If you haven't deployed a New Relic open source agent yet, [install New Relic](https://docs.newrelic.com/docs/agents/manage-apm-agents/installation/install-agent) for your application. Also, [install the Terraform CLI](https://www.terraform.io/intro/getting-started/install.html).
+
+Start by initializing a working directory:
+
+You will also need to install Terragrunt to follow this guide:
+```bash
+brew install terragrunt
+```
+
+To follow the examples in this gude, the accompanying example code is available on [GitHub](https://github.com/jsbnr/nr-terragrunt-intro-example)
+
+```bash
+git clone https://github.com/jsbnr/nr-terraform-intro-example.git
+```
+
+
+<Steps>
+
+<!-- TTERRAGRUNT CONFIGURATION -->
+
+<Step>
+
+## Create a new configuration
+
+Terragrunt is a wrapper that provides extra tooling for your Terraform configurations. In the next steps, you will create a new configuration using Terragrunt, Terraform, and New Relic.
+
+Start by initializing a working directory:
+```bash
+mkdir terragrunt-config && cd terragrunt-config
+```
+In the new directory, create a `terragrunt.hcl` file, create a  ./src directory, and create a `./evironments` directory:
+```bash
+touch terragrunt.hcl
+mkdir src
+mkdir environments
+```
+Inside of your __environments__ folder, create a folder named __dev.__
+In the `./src` directory, create a __main.tf__ and __provider.tf__ fiiles:
```

Comment:
```suggestion
In the `./src` directory, create a `main.tf` and `provider.tf` files:
```

Same here. Should these also use inline code to denote the files (this feedback would apply to the whole file)?

Also spotted a typo here